numeric.c

```diff
@@ -2548,26 +2545,38 @@ flo_to_i(VALUE num)
 
 /*
  *  call-seq:
- *     float.truncate([ndigits])  ->  integer or float
+ *    truncate(ndigits = 0) -> float or integer
  *
- *  Returns +float+ truncated (toward zero) to
- *  a precision of +ndigits+ decimal digits (default: 0).
+ *  Returns +self+ truncated (toward zero) to
+ *  a precision of +ndigits+ decimal digits.
  *
- *  When the precision is negative, the returned value is an integer
- *  with at least <code>ndigits.abs</code> trailing zeros.
+ *  When +ndigits+ is positive, returns a float with +ndigits+ digits
+ *  after the decimal point (as available):
  *
- *  Returns a floating point number when +ndigits+ is positive,
- *  otherwise returns an integer.
+ *    f = 12345.6789
```

Comment:
It's best to use a negative number here to show the difference between `floor` and `truncate`.  Instead of using 10 examples with  the same receiver and separate arguments for each of `floor`, `ciel`, `round`, and `truncate`, use one positive (12345.6789) and one negative receiver (-12345.6789) , and have 4 examples per receiver, using 1, 3, 0, and -3 as the arguments.

Reply:
Done.